atm.js

```diff
@@ -1,21 +1,101 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function(){
```

Comment:
This works but is unnecessary, as the DOM will have loaded by the time this script is called just before the closing body tag.